src/switch_core_media.c

```diff
@@ -6022,6 +6023,7 @@ SWITCH_DECLARE(uint8_t) switch_core_media_negotiate_sdp(switch_core_session_t *s
 
 					crypto = attr->a_value;
 					crypto_tag = atoi(crypto);
+					switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG, "Checking incoming crypto\n");
```

Comment:
I think we should add more info on this log such as the result on the switch_core_session_check_incoming_crypto

Reply:
If result is negative warning are printed as normally. Apart from that I added some logging inside switch_core_session_check_incoming_crypto. To be honest the purpose of this one is just to tell when the switch_core_session_check_incoming_crypto is executed.